api/lagoon/client/_lgraphql/projects/addExistingProjectToOrganization.graphql

```diff
@@ -0,0 +1,11 @@
+mutation (
```

Comment:
remove this mutation, and any associated machinery functionality tied to this mutation.

the underlying mutation is not entirely safe to run as it can move things in without considering other possibilities, the plan is to replace it with https://github.com/uselagoon/lagoon/pull/3598